DopplerJobsServer/Program.cs

```diff
@@ -2,7 +2,7 @@
 using Microsoft.Extensions.Hosting;
 using Serilog;
 
-namespace Doppler.Service.Job.Server
+namespace Server
```

Comment:
I think that just `Server` is very generic namespace. Could use a more specific namespace?

Reply:
`Doppler.Job.Server` is more clear, I'll do